src/components/RouteRow.tsx

```diff
@@ -31,12 +31,22 @@ export const RouteRow = ({ route }: Props) => {
   };
 
   return (
-    <Box width='100%'>
-      <Pressable onPress={navToRoute} >
-        <Flex flexDirection="row" justifyContent="space-between" >
-          <Text>{data.name}</Text>
-          <Text>{data.gradeDisplayString}</Text>
-        </Flex>
+    <Box>
+      <Pressable onPress={navToRoute}>
+        <HStack justifyContent='space-between'>
+          <Text p='2' fontSize={{
+            base: 'md',
+            lg: 'xl',
+            xl: '3xl'
+          }}>{data.name}</Text>
+          <Box justifyContent='center'>
+            <Text p='2' fontSize={{
+              base: 'md',
+              lg: 'xl',
+              xl: '3xl'
+            }}>{data.gradeDisplayString}</Text>
```

Comment:
Sorry I wasn't clear in the last comment, I think for each option the size should be one step down from the label size:
```ts
base: 'md',
lg: 'lg',
xl: '2xl'
```

Reply:
They were already 1 size smaller than the label's size. Added your change either way